packages/info/README.md

```diff
@@ -9,22 +9,79 @@ This package returns a set of information related to the local environment.
 ## Installation
 
 ```bash
+#npm
 npm i -D @webpack-cli/info
+
+#yarn
+yarn add @webpack-cli/info -D
+
+#npx
+npx webpack info [options]
+
 ```
 
 ## Usage
 
+### Args / Flags
+
+#### Output format
+
+| Flag                | Description                   | Type        |
+| ------------------- | ----------------------------- | ----------- |
+| `--output-json`     | To get the output as JSON     | [ boolean ] |
+| `--output-markdown` | To get the output as markdown | [ boolean ] |
+
+#### Options
+
+| Flag                | Description                                                     | Type        |
+| ------------------- | --------------------------------------------------------------- | ----------- |
+| `--help`            | Show help                                                       | [ boolean ] |
+| `--version`         | Show version number of `webpack-cli`                            | [ boolean ] |
+| `--system` , `-s`   | System information ( OS, CPU )                                  | [ boolean ] |
+| `--binaries` , `-b` | Installed binaries (Node, yarn, npm)                            | [ boolean ] |
+| `--browsers`        | Installed web browsers                                          | [ boolean ] |
+| `--npmg`            | Globally installed NPM packages ( webpack & webpack-cli only )  | [ boolean ] |
+| `--npmPackages`     | Info about packages related to webpack installed in the project | [ boolean ] |
+
 ### Node
 
 ```js
-const envinfo = require("@webpack-cli/info").default;
-envinfo();
+const info = require('@webpack-cli/info').default;
+
+async function wrapperFunc() {
+    await info({
+        /* Custom Config */
```

Comment:
what's this for

Reply:
You mean `Custom Config`?
If you pass an object as per the defined schema then you can get info about those parts only, omitting the defaults.

Reply:
Thanks for elaborating!